jacodb-ets/src/main/kotlin/org/jacodb/ets/model/EtsClass.kt

```diff
@@ -33,7 +33,7 @@ class EtsClassImpl(
     override val methods: List<EtsMethod>,
     override val ctor: EtsMethod,
 ) : EtsClass {
-    override var superClass: EtsClass? = null
+    override var superClass: EtsClassSignature? = null
```

Comment:
Since `superClass` is now just a signature, we can move it in the constructor and make immutable.